packages/front/src/components/b-task-expanded.vue

```diff
@@ -0,0 +1,229 @@
+      <BComment 
+        v-for="(action, index) in task.actions"
+        :author="action.user"
+        :content="action.content"
+        :type="action.type"
+        :hidden="action.currentRound"
+      />
+    </div>
+
+    <div class="b-task-expanded__wrapper" v-if="finished">
+      <BInputField
+        color="gray-30"
+        label="Write a comment"
+        name="newMessage"
+      >
+        <BTextArea 
+          name="newMessage"
+          @keyup.enter="comment"
+        />
+      </BInputField>
+    </div>
+
+    <div class="b-task-expanded__wrapper" v-if="finished">
+      <div class="b-task-expanded__card-container">
+        <BCard 
+          v-for="fibo in fibonacci"
+          :active="votable"
+          :value="fibo"
+          @click="vote(fibo)"
+        />
+      </div>
+    </div>
+    
+  </div>
+</template>
+
+<script>
+import { useStore } from 'vuex';
+import { api } from '../services/api';
+
+import BButton from '../components/b-button.vue';
+import BCard from '../components/b-card.vue';
+import BComment from '../components/b-comment.vue';
+import BInputField from '../components/b-input-field.vue';
+import BText from '../components/b-text.vue';
+import BTextArea from '../components/b-text-area.vue';
+
+export default {
+  name: 'BTaskExpanded',
+
+  components: {
+    BButton,
+    BCard,
+    BComment,
+    BInputField,
+    BText,
+    BTextArea,
+},
+
+  props: {
+    taskId: {
+      type: String,
+      required: true,
+    },
+    squadId: {
+      type: String,
+      required: true,
+    },
+  },
+
+  data() {
+    return {
+      task: {},
+      fibonacci: [1, 2, 3, 5, 8, 13],
+      votable: true,
+      finished: false,
+    }
+  },
+
+  computed: {
+    userEmail() {
+      return useStore().getters.getUserEmail;
+    },
+  },
+
+  methods: {
+    async vote(point) {
+      if(this.votable){
+        await api.post(`/squad/${this.squadId}/task/${this.taskId}/vote`, { points: point })
+          .then(console.log('success!'))
+          .catch((err) => {
+            console.log(err.response.data.message);
+          });
+        this.load();
+      }
+    },
+    async comment() {
+      await api.post(`/squad/${this.squadId}/task/${this.taskId}/message`, { message: `${newMessage.value}` })
+        .then(console.log('success!'))
+        .catch((err) => {
+          console.log(err.response.data.message);
+        });
+      this.load();
+    },
+    async load() {
+      await api.get(`/squad/${this.squadId}/task/${this.taskId}`)
+        .then((res) => {
+          this.task = res.data;
+          this.finished = !(this.task.finished);
+          this.task.actions.every(x => this.votable = this.eligible(x));
+        })
+        .catch((err) => {
+          console.log(err.response.data.message);
+        });
+      
+    },
+    eligible(x) {
+      if(x.type == "vote" && x.currentRound==true && x.email==this.userEmail){
+        return false;
```

Comment:
Aqui dá pra deixar só `return !(x.type == "vote" && x.currentRound==true && x.email==this.userEmail)`

Reply:
[mudei aqui](https://github.com/AsyncPlanningPoker/BrieflyPlanningPoker/pull/60/commits/5cdebea29caad7c0bf84f7adb76452f8968cfb45) pincesa